src/ui/components/Library/Team/View/Tests/Overview/TestDetails.tsx

```diff
@@ -1,13 +1,107 @@
-import { TestExecution } from "shared/test-suites/TestRun";
+import orderBy from "lodash/orderBy";
+
+import { GroupedTestRun, TestExecution } from "shared/test-suites/TestRun";
+import MaterialIcon from "ui/components/shared/MaterialIcon";
 import { testFailed, testPassed } from "ui/utils/testRuns";
 
-import { ReplayList } from "./ReplayList";
+import { getTruncatedRelativeDate } from "../../Recordings/RecordingListItem/RecordingListItem";
+import { StatusIcon } from "./StatusIcon";
+
+export function TestDetails({ testRuns }: { testRuns: GroupedTestRun[] }) {
+  const sortedTestRuns = orderBy(testRuns, "date", "desc");
+
+  return (
+    <div className="flex flex-col gap-1 p-2">
+      {sortedTestRuns.map((t: any) => (
+        <TestRun testRun={t} key={t.testRunId} />
+      ))}
+    </div>
+  );
+}
+
+function TestRun({ testRun }: { testRun: GroupedTestRun }) {
+  const executionsWithReplays = testRun.executions.filter(
+    e => e.recordings.length && (testPassed(e) || testFailed(e))
+  );
+
+  if (!executionsWithReplays.length) {
+    return null;
+  }
+
+  const sortedExecutions = orderBy(executionsWithReplays, "createdAt", "desc");
+  const lastReplay = sortedExecutions[0];
+  const remainingReplays = sortedExecutions.slice(1);
+
+  return (
+    <div className="flex flex-col rounded-md bg-chrome">
+      <FinalAttemptReplay
+        execution={lastReplay}
+        title={lastReplay.commitTitle || "<Commit title unavailable>"}
+      />
+      {remainingReplays.map((e: any, i: any) => (
+        <NonFinalAttemptReplay
+          key={i}
+          execution={e}
+          title={
+            i == remainingReplays.length - 1
+              ? "Initial attempt"
+              : `Retry ${remainingReplays.length - 1 - i}`
+          }
+        />
+      ))}
+    </div>
+  );
+}
```

Comment:
This is what's being rendered. Is this what it should look like?
![Screenshot 2023-12-04 at 3 27 32 PM](https://github.com/replayio/devtools/assets/29597/6410da85-3a30-4494-b6d1-d6d396220b39)

Looks a little odd to me.
* Top row has more vertical padding than the others
* Top row renders as a flex column but the others render as flex rows, which I think would look weirder if the title was short:
![Screenshot 2023-12-04 at 3 28 40 PM](https://github.com/replayio/devtools/assets/29597/e27d68ae-cd4f-483b-a1f1-a692a6e776bf)

Making the padding and flex direction more consistent helps a bit I think but then the info seems a bit confusingly presented:
![Screenshot 2023-12-04 at 3 31 08 PM](https://github.com/replayio/devtools/assets/29597/fdccac3c-e4cb-4c8c-8fb4-e1dffdf13af9)

I wonder if it wouldn't be clearer to pull the commit title out so it's separate from the retries?
![Screenshot 2023-12-04 at 3 33 17 PM](https://github.com/replayio/devtools/assets/29597/5541ee38-1b0a-4dff-9383-b1d484fd15b8)

I haven't seen the UI spec for this but I'm wondering how we landed on the spect that's implemented here.

Reply:
The original design:
<img width="811" alt="image" src="https://github.com/replayio/devtools/assets/15959269/7a166f6f-b806-4859-b9c1-92915634e959">

Reply:
@jonbell-lot23 I made some (light) alterations here from the original spec, but I think something bigger that I've noticed and Brian's also pointed out is that the way the information's presented here is somewhat confusing.

In this example:
![image](https://github.com/replayio/devtools/assets/15959269/d355ea3d-ba1d-40ef-9567-b3e7bbffcef2)

It's not clear to me that there were 4 distinct attempts for this test run, and the link to open the latest one is the first row with "Merge 961b…". I think what would be clearer is to group by the test run, then have all four attempts displayed clearly below. For example, for a test run that had 3 attempts, it would look something like

```
-------------------
My commit message goes here
3h ago
-------------------
▶️ Attempt 2
▶️ Attempt 1
▶️ Initial attempt
-------------------
```

Reply:
Yup, this sounds good to me!

Reply:
Some important info seems to have gotten dropped between Jon's mock and the "alterations" –
* The top row should have the user who triggered the run
* The top row should have the the time when the (last?) test attempt was
* Individual rows should have _duration_ not _time ago_ times

Reply:
Yeah, I should clarify my thumbs up above.

I'm saying the idea of putting metadata up top and attempts below that seem good to me. I have not caught up on the whole discussion and I am sure there are other details that I can find once a do more a more complete NUX pass. Bear with me, I have a lot to catch up on after my concussion.